roles/sahara/meta/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+dependencies:
+  - role: openstack-database
+    database_name: sahara
+    package_alternatives: "{{ sahara.package.console_scripts }}"
```

Comment:
I think you referring to package_alternatives: "{{ sahara.package.console_scripts }}". I did used it earlier but then I removed it. Looks like committed by mistake.